src/scripts/main.js

```diff
@@ -22,7 +22,21 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.append(list);
+
+  for (const key in data) {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+    list.append(item);
+
+    if (typeof data[key] === 'object'
```

Comment:
https://prnt.sc/u50qcr
useless 'ul' elements, 
I guess, this happened because an empty object is still an object.

Reply:
thanks! changed